src/avfoundation.cs

```diff
@@ -11481,7 +11481,14 @@ interface AVCaptureDeviceFormat {
 
 		[iOS (8,0), NoMac]
 		[Export ("videoHDRSupported")]
+		bool VideoHDRSupportedVideoHDREnabled { [Bind ("isVideoHDRSupported")] get; }
+
+#if !XAMCORE_5_0
+		[iOS (8,0), NoMac]
+		[Obsolete ("Use the 'VideoHDRSupportedVideoHDREnabled' property instead.")]
+		[Wrap ("VideoHDRSupportedVideoHDREnabled", IsVirtual = true)]
```

Comment:
Im sure you would have done this anyways, but just a reminder to change the method name in the Wrap if you do Alex's suggestion :)